DRUM/README.md

```diff
@@ -0,0 +1,24 @@
+# DRUM
+
+*This web app allows you to make your own music with all the beats that are present in the app. You can click on the alphabets or the images that are present on the screen and enjoy the beats. You can also use your keyboard keys to make the music.*
+>Technologies used:
+- HTML
+- CSS
+- JAVASCRIPT
+
+#### Steps to Use:
+
+---
+
+- Download or clone the repository
+
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+
+- Go to the directory
+- Run the index.html file
+- Start Using The DRUM
+## Relevant Screenshots of the application
+
+![drum](images/drumkit.PNG)
```

Comment:
THe screenshot is not working.

Reply:
the screenshot is visible i guess.
![image](https://user-images.githubusercontent.com/54364704/125208689-ae856a80-e2b1-11eb-82f4-f39aa59f2814.png)

Reply:
@ayushseth07 No it isn't. Please use a merged PR for reference.